setup.py

```diff
@@ -144,5 +144,9 @@ def pkgPath(root, path, rpath="/"):
     scripts=['bin/diamond', 'bin/diamond-setup'],
     data_files=data_files,
     install_requires=install_requires,
+    classifiers=[
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        ],
```

Comment:
I tried it and that broke the regression tests, so I reverted the indentation level back.